cx_Freeze/freezer.py

```diff
@@ -221,9 +221,10 @@ def _GetDefaultBinIncludes(self):
         """Return the file names of libraries which must be included for the
            frozen executable to work."""
         if sys.platform == "win32":
-            pythonDll = "python%s%s.dll" % sys.version_info[:2]
-            return [pythonDll, "gdiplus.dll", "mfc71.dll", "msvcp71.dll",
-                    "msvcr71.dll"]
+            pythonDlls = ["python%s.dll" % sys.version_info[0],
+                          "python%s%s.dll" % sys.version_info[:2],
+                          "vcruntime140.dll"]
+            return pythonDlls
```

Comment:
Why drop gdiplus.dll, mfc71.dll, msvcp71.dll and msvcr71.dll? Why not just add vcruntime140.dll? And how does this work with include_mscvcr? I'm not a regular Windows user so can you explain what the intent of this is? Thanks!

Reply:
Py3 do not use this dlls.